docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -152,127 +137,42 @@ system.time({
-  # within each parallel task, random values for input parameters (cash flows and discount rate) are generated for each trial
-  # these random input values represent different possible scenarios
-  results <- foreach(i = 1:num_trials, .combine = rbind) %dopar% {
-    # randomly generate input values for each trial
-    cashflows <- runif(10000, min = -100, max = 100)  # random cash flow vector over 10,000 time periods.
-    # these cash flows can represent costs (e.g., initial investment) and benefits (e.g., revenue or savings) associated with the project
-    discount_rate <- runif(1, min = 0.05, max = 0.15)  # random discount rate at which future cash flows are discounted
+!!! Notes
+    If you have not installed `foreach` and `doParallel` packages. You have to run these commands after loading `r`: `install.packages("foreach")` and `install.packages("doParallel")`
 
-    # calculate NPV for the trial
-    npv <- npv_calculation(cashflows, discount_rate)
+Once the program is running, monitor your usage with `userload` command in the second window:
 
-  }
-})
-cat("Parallel NPV Calculation (using", ncore, "cores):\n")
-# print summary statistics for NPV and plot a histogram of the results
-# positive NPV indicates that the project is expected to generate a profit (the benefits outweigh the costs),
-# making it an economically sound decision. If the NPV is negative, it suggests that the project may not be financially viable.
-summary(results)
-hist(results, main = 'NPV distribution')
+```bash title="Terminal 2 Command"
+watch -n 1 userload
 ```
 
-Now, we can run this script with varying number of cores. We will still limit the number of cores to 48 on yen[2-5] and to 12 cores on yen1 per
-<a href="/yen/community.html" target="_blank">Community Guidelines</a>.
+While the program is running, you should see about 8 CPU cores being utilized in `userload` output. Note that the `watch` command allows you to see the real time CPU cores count every 1 second.
 
-For example, to run with 12 cores:
+![Resource monitoring during parallel run](/assets/images/monitor_userload_r_8_cores.png)
 
-```bash
-$ Rscript investment-npv-parallel-args.R 12
-```
+Run `htop -u $USER` in the third window, where `$USER` is your SUNetID:
 
-You should see:
-```bash
-Loading required package: iterators
-Loading required package: parallel
-   user  system elapsed
-302.366   2.344 116.261
-Parallel NPV Calculation (using 12 cores):
-       V1
- Min.   :-682.9972
- 1st Qu.: -96.5376
- Median :  -0.2428
- Mean   :  -0.2369
- 3rd Qu.:  96.1112
- Max.   : 720.2979
+```bash title="Terminal 3 Command"
+htop -u $USER
 ```
 
-Monitor your CPU usage while the program is running in the other terminal window with `htop` and `userload`.
-
+While the program is running, you should see 8 R processes running in the `htop` output because we specified 8 cores in our R program.
+![resource monitoring during parallel run](/assets/images/monitor_htop_r_8_cores.png)
```

Comment:
Maybe we add a conclusion sentence to the end of the page. It sort of ends abruptly.